src/components/seeds/database/columns.ts

```diff
@@ -389,12 +404,19 @@ const germinationTestingPreset: Preset = {
   ],
 };
 
-export const searchPresets = [
-  defaultPreset,
-  generalInventoryPreset,
-  seedStoragePreset,
-  viabilitySummaryPreset,
-  germinationTestingPreset,
-];
+export const searchPresets = (preferredWeightSystem: string) => {
+  const weightUnitsEnabled = isEnabled('Weight units');
+
+  if (weightUnitsEnabled && preferredWeightSystem === 'imperial') {
```

Comment:
if you use my suggestion above, this gets simpler.. and only one list to maintain